charts/housnav/templates/deployment.yaml

```diff
@@ -13,14 +13,32 @@ spec:
     metadata:
       labels:
         app: house-innovations-housnav-web
+      annotations:
+        redeploy-timestamp: {{ now | quote }}
+        vault.hashicorp.com/agent-inject: 'true'
+        vault.hashicorp.com/agent-inject-secret-secrets.env: {{ .Values.vault.secretPath }}
+        vault.hashicorp.com/agent-inject-template-secrets.env: |
+          {{`{{- with secret "`}}{{ .Values.vault.secretPath }}{{`" -}}
+          {{- range $key, $val := .Data.data }}
+          export {{ $key }}={{ $val }}{{ end -}}
+          {{- end -}}`}}
+        vault.hashicorp.com/agent-pre-populate-only: 'true'
+        vault.hashicorp.com/agent-inject-token: 'false'
```

Comment:
Is there a reason we don't want to inject the token?

Reply:
The token is needed if the application dynamically takes the secret directly from the Vault. In our case, the application takes the necessary parameters from the container environment variable. The token injection is redundant at the moment.